src/main/java/org/first5924/frc2023/commands/vision/UpdatePoseEstimator.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
This seems like overkill for a single command. Let's pull this in as a function of `DriveSubsystem`

Reply:
Resolved by running in periodic method of DriveSubsystem